src/common/line_entry.rs

```diff
@@ -72,6 +72,56 @@ impl LineEntry {
         }
         comment::parse(self.raw_string.as_str())
     }
+
+    #[allow(dead_code)]
+    pub fn get_substitution_key(&self) -> Vec<&str> {
+        let mut keys = Vec::new();
+
+        let mut value = match self.get_value() {
+            Some(value) => value.trim(),
+            None => return keys,
+        };
+
+        if value.starts_with('\'') {
+            return keys;
+        }
```

Comment:
This can be combined to one match 🙂 
```suggestion
        let mut value = match self.get_value().map(|v| v.trim()) {
            Some(value) if !value.starts_with('\'') => value,
            _ => return keys,
        };
```

Reply:
Done, thank you!